src/test/java/ca/bc/gov/open/Scss/CourtControllerTests.java

```diff
@@ -0,0 +1,237 @@
+                .thenReturn(responseEntity);
+
+        //     Do request
+        var out = courtController.getCeisConnectInfo(ci);
+
+        //     Assert response is correct
+        assert cir.equals(out);
+    }
+
+    @Test
+    public void getPartiesTest() {
+        //  Init service under test
+        courtController = new CourtController(restTemplate);
+
+        //    Init request object
+        var gp = new GetParties();
+        gp.setPhysicalFileId(BigDecimal.ONE);
+
+        //    Init response
+        var cb = new GetPartiesResponse();
+        CaseParty cp = new CaseParty();
+        cb.setParties(Collections.singletonList(cp));
+        ResponseEntity<GetPartiesResponse> responseEntity = new ResponseEntity<>(cb, HttpStatus.OK);
+
+        //     Set up to mock ords response
+        when(restTemplate.exchange(
+                        Mockito.any(String.class),
+                        Mockito.eq(HttpMethod.GET),
+                        Mockito.<HttpEntity<String>>any(),
+                        Mockito.<Class<GetPartiesResponse>>any()))
+                .thenReturn(responseEntity);
+
+        //     Do request
+        var out = courtController.getParties(gp);
+
+        //     Assert response is correct
+        assert (out.equals(cb));
+    }
+
+    @Test
+    public void partNameSearchTest() {
+        //  Init service under test
+        courtController = new CourtController(restTemplate);
+
+        //    Init request object
+        var pns = new PartyNameSearch();
+        PartyNameSearchFilter pnf = new PartyNameSearchFilter();
+        pnf.setCourtClass("A");
+        pnf.setAgencyId(BigDecimal.ONE);
+        pnf.setSearchType("A");
+        pnf.setFirstName("A");
+        pnf.setCourtLevel("A");
+        pnf.setPage(BigDecimal.ONE);
+        pnf.setName("A");
+        pnf.setRoleType("A");
+        pns.setFilter(pnf);
+
+        //    Init response
+        SearchResults res = new SearchResults();
+
+        ResponseEntity<SearchResults> responseEntity = new ResponseEntity<>(res, HttpStatus.OK);
+
+        //     Set up to mock ords response
+        when(restTemplate.exchange(
+                        Mockito.any(String.class),
+                        Mockito.eq(HttpMethod.GET),
+                        Mockito.<HttpEntity<String>>any(),
+                        Mockito.<Class<SearchResults>>any()))
+                .thenReturn(responseEntity);
+
+        //     Do request
+        var out = courtController.partyNameSearch(pns);
+
+        //     Assert response is correct
+        assert (out.getSearchResults().equals(res));
+    }
+
+    @Test
+    public void saveHearingResultTest() {
+        //  Init service under test
+        courtController = new CourtController(restTemplate);
+
+        ResponseEntity<String> responseEntity = new ResponseEntity<>("", HttpStatus.OK);
+
+        var hr = new SaveHearingResults();
+        //    var res = new HearingResult();
```

Comment:
If this is needed lets fill out and get it working.

Reply:
Filled out